Source/Particles/Collision/MCCScattering.H

```diff
@@ -195,6 +196,7 @@ public:
     *
     * @param[in] energy_cost energy cost of ionization
     * @param[in] mass1 mass of the colliding species
+    * @param[in] ion_vel_std ...?
```

Comment:
@roelof-groenewald @KZhu-ME maybe? :)

Reply:
That parameter is to capture the ion temperature:
```suggestion
    * @param[in] ion_vel_std standard deviation of ion velocity distribution; for
    *                        a Maxwellian distribution it should equal sqrt(kB*T/m)
```